### ISSUE TITLE
ability to use string data type for all parquet columns

### ISSUE THREAD

**Issue by tooptoop4** [kindly/flatterer#62]
would be great to have option to write all columns as STRING data type in the final parquet, instead of int..etc

**Comment by kindly** [comment]
yes I will consider adding this as it will make things faster as there will be no need for type detection for this case.

**Comment by kindly** [comment]
This option has been added.

**Comment by tooptoop4** [comment]
changelog says 2024

**Comment by kindly** [comment]
@tooptoop4 This is fixed now
